### ISSUE TITLE
limits_conf "unlimitied" error

### ISSUE THREAD

**Issue by csams** [RedHatInsights/insights-core#447]
Many archives have limit.conf files with values of "unlimitied" instead of "unlimited."

I think the parser did the right thing by raising an exception on a value that wasn't valid, but now that we've seen it (a lot), we need to handle it gracefully and maybe write a rule for it.

```
2017-09-17 18:43:02 ERROR  invalid literal for int() with base 10: 'unlimitied'
Traceback (most recent call last):
  File "build/bdist.linux-x86_64/egg/insights/core/plugins.py", line 34, in parser_executor
    r = component(d)
  File "build/bdist.linux-x86_64/egg/insights/core/__init__.py", line 89, in __init__
    self.parse_content(context.content)
  File "build/bdist.linux-x86_64/egg/insights/parsers/limits_conf.py", line 76, in parse_content
    value = int(value)
ValueError: invalid literal for int() with base 10: 'unlimitied'
```

**Comment by PaulWay** [comment]
Good point.  I think this needs a separate parser that just looks for invalid configuration like that.

**Comment by PaulWay** [comment]
Also - we're getting "a lot" of customers mistyping this?  I wonder where that's coming from?  I searched Google for site:access.redhat.com unlimitied and didn't find anything, and searched for 'unlimitied' in the knowledge base and only found one article that didn't relate to limits.conf.  Is it misspelled in some part of our standard configuration?  Is it coming from something we've written?

**Comment by PaulWay** [comment]
A third point: is this a fatal error in limits.conf or does PAM simply ignore the misspelling?  Maybe instead of a parse exception we need to simply flag it as an invalid line or something.  It'd be much easier to write a parser in that case.

**Comment by csams** [comment]
Good questions. I also searched around after I first came across the issue but couldn't find anything. At least several hundred archives have it (maybe 1000+), but I don't know how many individual users it affects (I suspect just one with a lot of machines). I'm also not sure how the system reacts to the bad value, although the man page says "The pam_limits module does report configuration problems found in its configuration file and errors via syslog(3)."

**Comment by PaulWay** [comment]
If it doesn't break PAM, then let's just catch it in a list of 'bad lines' and deal with it there.

**Comment by csams** [comment]
Sounds good to me.

**Comment by csams** [comment]
1.x fixed in #589

**Comment by PaulWay** [comment]
I think these are now resolved in https://github.com/RedHatInsights/insights-core/pull/589 and https://github.com/RedHatInsights/insights-core/pull/590
